v2/internal/pkg/operator/catalog_handler.go

```diff
@@ -418,28 +441,29 @@ func isPreReleaseOfFilteredVersion(version string, chEntryName string, filteredV
 	return false
 }
 
-func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) []v2alpha1.RelatedImage {
+func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) ([]v2alpha1.RelatedImage, error) {
 	var relatedImages []v2alpha1.RelatedImage
 
 	for _, ri := range bundle.RelatedImages {
 		if strings.Contains(ri.Image, "oci://") {
-			internalLog.Warn("%s 'oci' is not supported in operator catalogs : SKIPPING", ri.Image)
-			continue
+			msg := fmt.Sprintf("Image: %s Bundle: %s Operator: %s 'oci' is not supported in operator catalogs : SKIPPING", ri.Image, bundle.Name, operatorName)
```

Comment:
```suggestion
			msg := fmt.Sprintf("image: %s Bundle: %s Operator: %s 'oci' is not supported in operator catalogs", ri.Image, bundle.Name, operatorName)
```
We are not really skipping. And I think it's better not to capitalize error messages since they can be wrapped in other errors.